### ISSUE TITLE
GitHub to Guru: FOOTER not working for INTERNAL collections

### ISSUE THREAD

**Issue by shelbygluck** [peckjon/github-to-guru#25]
Add Footer optional parameter is not performing (or I am misunderstanding something about the configuration of this parameter).

My workflow

----
```
name: Create guru cards

on:
  push:

jobs:
  guru:
    runs-on: ubuntu-latest
    steps:
    - uses: actions/checkout@v1
    - uses: peckjon/github-to-guru@master
      env:
        GURU_USER_EMAIL: '${{ secrets.GURU_USER_EMAIL }}'
        GURU_USER_TOKEN: '${{ secrets.GURU_USER_TOKEN }}'
        GURU_COLLECTION_ID: '********************'
        GURU_CARD_FOOTER: 'This information is maintained by SEs in GitHub, at x repo. To request an edit on this card, message the original author listed on the corresponding document'
        GURU_CARD_YAML: ADD_TO_GURU.yaml
```
----

The card is being successfully created in Guru without the footer text included anywhere.

**Comment by peckjon** [comment]
Thanks @shelbygluck !  Your config looks correct.

Do you know if your collection is "internal" or "external"? There should be a line in the logfile of your workflow run: `Found {EXTERNAL | INTERNAL} collection at https://app.getguru.com/collections/...`

If it is EXTERNAL, this should be working, so there's a bug I need to track down.

If it is INTERNAL ... looks like I [didn't implement](https://github.com/peckjon/github-to-guru/blob/master/index.js#L285) the GURU_CARD_FOOTER option for that collection type (as [here](https://github.com/peckjon/github-to-guru/blob/master/index.js#L114)). That may have been an oversight, or there may be a technical reason. But let me know and I'll see if I can create an update that works.

**Comment by peckjon** [comment]
Just in case it's the latter, I've created a [potential fix](https://github.com/peckjon/github-to-guru/pull/26/commits/371834f6e27342c38913e40d88d98d31fb9d737f). I don't currently have access to a test env, but you should be able to try this branch by changing this line of your workflow:

```yaml
    - uses: peckjon/github-to-guru@master
```

To one of the following:

```yaml
    - uses: peckjon/github-to-guru@peckjon-footer-for-internal
```

```yaml
    - uses: peckjon/github-to-guru@371834f6e27342c38913e40d88d98d31fb9d737f
```

Please let me know if this works!

**Comment by maherio** [comment]
Hey I ran into this same issue, and I confirmed my collection is `INTERNAL`. I'm also using the single file `GURU_CARD_YAML` config option.

When attempting to use your bugfix branch, i got the following error:

```
Error: Unable to prepare card: Assignment to constant variable.
```

**Comment by peckjon** [comment]
Gah, missed that const. Thanks! Should be good to test now?

**Comment by maherio** [comment]
looks like a [different line](https://github.com/peckjon/github-to-guru/pull/26/files#r1621462419) needs to be changed

**Comment by peckjon** [comment]
@maherio ❤️ THANK YOU for your diligence and patience with this. I'm obviously working too quickly and not giving this proper attention (insufficient time, yesterday's commit was on mobile).

I've revised and I think mashed the `const` bugs (and also eliminated `var` entirely for `let`). COuld you give it another go?

Also, would you be interested in being brought in as a maintainer on this project?

**Comment by maherio** [comment]
It worked! Thanks!

Unfortunately I just can't commit any spare time right now, feel free to @ me in the future but I know I won't be responsive for the foreseeable future

**Comment by peckjon** [comment]
Completely understand. Thank you for your help on this one @maherio !

@shelbygluck I'm merging the fix #26 into main and it is now available as `peckjon/github-to-guru@1.9.0`
